From 0a6b7bf7ae0c23085ba5d28cefa1e3269bc54121 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 23 Jan 2022 07:38:32 +0000
Subject: [PATCH 5/5] WinSystemGbmGLESContext: wait longer for vsync

Waiting longer for vsync improves idle CPU consumption on ARM devices

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp b/xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp
index 414dc36963..771747ccbe 100644
--- a/xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp
+++ b/xbmc/windowing/gbm/WinSystemGbmGLESContext.cpp
@@ -147,7 +147,7 @@ void CWinSystemGbmGLESContext::PresentRender(bool rendered, bool videoLayer)
   }
   else
   {
-    KODI::TIME::Sleep(10ms);
+    KODI::TIME::Sleep(40ms);
   }
 }
 
-- 
2.17.1

